includes/class-sensei-customizer.php

```diff
@@ -23,15 +23,15 @@ class Sensei_Customizer {
 	 */
 	public function __construct() {
 		$this->colors = [
-			'sensei-course-theme-primary-color'    => [
+			'sensei-learning-mode-primary-color'    => [
```

Comment:
These are stored as options, so we'd also need to add a migration here. Otherwise the colors the user set would be lost.

Reply:
Done here 99424ac8bb092cc3678c0c2d96dd4193047eac21. I have ensured the old options are copied to the new options but the customisations are not reflecting on the frontend. Any idea why @yscik ?

Reply:
Hm, might be an unrelated change of the theme's global styles taking priority. Are the migrated colors showing up in the Customizer panel? If they are it should be good to go. (We'll tweak the style priority to fix it.)

Reply:
Oh yeah the migrated colors show on the customizer panel.